packages/mrc-ide-covidsim/src/main.ts

```diff
@@ -38,9 +35,7 @@ async function main() {
 
     // Read the request input JSON.
     const inputData = readFileSync(inputFilename, 'utf8')
-    const input = JSON.parse(inputData) as RequestInput
-    inputID = input.id
-    callbackURL = input.callbackURL
+    const input = JSON.parse(inputData) as input.ModelInput
```

Comment:
nit: A little confusing that you are overloading `input` here. It happens to work since the `input` from `import { input } from '@covid-modeling/api'` is being used only as a type, but I'd still try to avoid name overloading.

Maybe it's best if you export `ModelInput` directly from `api/src/index.ts`. I'll make a comment there, too.